depends/hosts/darwin.mk

```diff
@@ -1,7 +1,7 @@
 OSX_MIN_VERSION=11.0
-OSX_SDK_VERSION=11.0
-XCODE_VERSION=12.2
-XCODE_BUILD_ID=12B45b
+OSX_SDK_VERSION=14.0
+XCODE_VERSION=15.0
+XCODE_BUILD_ID=15A240d
```

Comment:
According to https://developer.apple.com/support/xcode/, additionally, `OSX_MIN_VERSION=13.5` should be adjusted as well.